fhireval/data/BasicTest.json

```diff
@@ -0,0 +1,328 @@
+					},
+					{
+					  "system": "https://www.ohdsi.org",
+					  "code": "3025315",
+					  "display": "Body weight"
+					}
+				  ],
+				  "text": "Body weight"
+				},
+				"subject": {
+				  "reference": "Patient/TEST-Patient-1"
+				},
+				"_effectiveDateTime": {
+				  "extension": [
+					{
+					  "url": "http://hl7.org/fhir/StructureDefinition/cqf-relativeDateTime",
+					  "extension": [
+						{
+						  "url": "target",
+						  "valueReference": {
+							"reference": "Patient/TEST-Patient-1"
+						  }
+						},
+						{
+						  "url": "targetPath",
+						  "valueString": "birthDate"
+						},
+						{
+						  "url": "relationship",
+						  "valueCode": "after"
+						},
+						{
+						  "url": "offset",
+						  "valueDuration": {
+							"value": 1.911,
+							"unit": "a",
+							"system": "http://unitsofmeasure.org",
+							"code": "years"
+						  }
+						}
+					  ]
+					}
+				  ]
+				},
+				"valueQuantity": {
+				  "value": 73.13492574157895,
+				  "unit": "kg",
+				  "system": "http://unitsofmeasure.org",
+				  "code": "kg"
+				}
+			  },
+			  {
+				"resourceType": "Observation",
+				"extension": [
+				  {
+					"url": "http://ncpi-fhir.github.io/StructureDefinition/visit-number",
+					"valueInteger": 1
+				  }
+				],
+				"identifier": [
+				  {
+					"system": "urn:ncpi-test:unique-string",
+					"value": "BasicTest-Observation-3"
+				  }
+				],
+				"status": "final",
+				"code": {
+				  "coding": [
+					{
+					  "system": "http://loinc.org",
+					  "code": "39156-5",
+					  "display": "Body mass index (BMI) [Ratio]"
+					},
+					{
+					  "system": "https://www.ohdsi.org",
+					  "code": "3038553",
+					  "display": "Body mass index (BMI) [Ratio]"
+					}
+				  ],
+				  "text": "Body mass index (BMI) [Ratio]"
+				},
+				"subject": {
+				  "reference": "Patient/TEST-Patient-1"
+				},
+				"_effectiveDateTime": {
+				  "extension": [
+					{
+					  "url": "http://hl7.org/fhir/StructureDefinition/cqf-relativeDateTime",
+					  "extension": [
+						{
+						  "url": "target",
+						  "valueReference": {
+							"reference": "Patient/TEST-Patient-1"
+						  }
+						},
+						{
+						  "url": "targetPath",
+						  "valueString": "birthDate"
+						},
+						{
+						  "url": "relationship",
+						  "valueCode": "after"
+						},
+						{
+						  "url": "offset",
+						  "valueDuration": {
+							"value": 1.911,
+							"unit": "a",
+							"system": "http://unitsofmeasure.org",
+							"code": "years"
+						  }
+						}
+					  ]
+					}
+				  ]
+				},
+				"valueQuantity": {
+				  "value": 26.621524665624275,
+				  "unit": "kg/m2",
+				  "system": "http://unitsofmeasure.org",
+				  "code": "kg/m2"
+				},
+				"derivedFrom": [
+				  {
+					"reference": "Observation/BasicTest-Observation-1"
+				  },
+				  {
+					"reference": "Observation/BasicTest-Observation-2"
+				  }
+				]
+			  }
+		],
+		"Extension": [ {
+			"resourceType": "StructureDefinition",
```

Comment:
I think we can use `Encounter` instead of creating a custom profile?

Reply:
This is an extension created for the purposes of testing that extensions work. It doesn't really matter if it's a particularly good solution for the real world. It's purpose in life is just to provide a way to verify that a given FHIR platform behaves as expected when using an extension. I'm happy to use something else if folks have suggestions, but this was an easy solution.